assets/css/v2/elevator/elevator_closures.scss

```diff
@@ -66,8 +66,7 @@
         transform: translateX(calc(-100% * var(--closure-list-offset)));
 
         .closure-row {
-          max-width: 1080px;
-          padding: 0 48px;
```

Comment:
Having a universal padding on a container feels more reliable than needing a (possibly duplicated) margin on each kind of item within the container, but maybe I'm overlooking a reason it has to be this way.

Reply:
I agree so I revisited. I think I came up with a better solution.